puppetboard/static/js/tables.js

```diff
@@ -6,16 +6,21 @@
 
   $(function() {});
 
-  if ($('th.default-sort').data()) {
-    $('table.sortable').tablesort().data('tablesort').sort($("th.default-sort"), "desc");
-  }
-
   $('thead th.date').data('sortBy', function(th, td, tablesort) {
     var tdTime = new Date(td.text().replace("-", ""));
     if(isNaN(tdTime)) return 0;
     else return tdTime;
   });
 
+  if ($('th.default-sort').data()) {
+    $.tablesort.DEBUG = false;
+    var tablesort = $('table.sortable').tablesort().data('tablesort');
+    // explicitly set the current index so we don't get default sort (asc)
```

Comment:
This comment is relevant to the commit but I don't think it's necessary in the source.

You should change as little as possible. Don't move line 9 to line 15 if you don't have to.

The commit should simply remove line 10 and replace it with:

```
var tablesort = $('table.sortable').tablesort().data('tablesort');
ablesort.index = $('th.default-sort').index();
tablesort.sort($("th.default-sort"), 'desc');
```

Reply:
The reordering of the two calls is necessary unfortunately.

As for the annotations, I just removed them and force pushed.